modules/store.js

```diff
@@ -0,0 +1,27 @@
+// Store Class: Handle storage - local storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
```

Comment:
- [Optonal] Good job on completing this task, I noticed that when I add a book, I do not know if it's added or not, which affects user experience, I would suggest after submitting the form input field should be set to empty, so as to give the user a sense of submitted.